### PR TITLE
update cell death curves for PBMCs

### PR DESCRIPTION
**Pull request by MikeLippincott**
This PR updates the cell death curves for PBMCs to include the 9hour time point.

### CODE REVIEW

File: figures/death_curves/figures/death_curve_zoomed.png
Comment (jenna-tomkinson) [inline figures/death_curves/figures/death_curve_zoomed.png:1]: Super nit-pick, you might consider making either media or DMSO a slightly darker grey to match the pattern with the other two treatments (LPS and Thaps).

File: figures/death_curves/scripts/deathcurves_plot.r [hunk 1]
```diff
@@ -15,32 +15,34 @@ death_df <- read.csv(death_curve_path)
 head(death_df)
 
 # death percentage
-death_df$percentage_dead_cells <- death_df$Death.Cells/(death_df$Death.Cells + death_df$Live.Cells)	*100
-death_df$treatment <- paste0(death_df$Compound," ",death_df$Dose," ",death_df$unit, " ",death_df$time, " ", death_df$`unit.1`)
-head(death_df)
+# drop the NA values
+death_df <- death_df[!is.na(death_df$Dead_cells),]
+death_df$percentage_dead_cells <- death_df$Dead_cells/(death_df$Dead_cells + death_df$Live_cells)	*100
```
Comment (jenna-tomkinson) [inline figures/death_curves/scripts/deathcurves_plot.r:20]:
```suggestion
death_df$percentage_dead_cells <- death_df$Dead_cells/(death_df$Dead_cells + death_df$Live_cells)*100
```
Not sure if the space there is necessary so consider removing it.

File: figures/death_curves/scripts/deathcurves_plot.r [hunk 2]
```diff
@@ -15,32 +15,34 @@ death_df <- read.csv(death_curve_path)
...
 death_df_agg$treatment <- factor(death_df_agg$treatment,levels = c(
     'Media 0 ',
-    'DMSO 0 ',
+    'DMSO 0.1 %',
     'LPS 1 ug/mL',
     'LPS 10 ug/mL',
-    'Thapsigargin 1 uM',
-    'Thapsigargin 10 uM'
+    'Thapsi 1 uM',
+    'Thapsi 10 uM'
```
Comment (jenna-tomkinson) [inline figures/death_curves/scripts/deathcurves_plot.r:35]: I don't see this name change in the plots, was this supposed to make a change for the figure or no?

**Review by jenna-tomkinson — APPROVED** [review]
LGTM! Nice short and concise PR!

File: figures/death_curves/figures/death_curve_zoomed.png
Comment (MikeLippincott) [reply figures/death_curves/figures/death_curve_zoomed.png:1]: Applied comment!

File: figures/death_curves/scripts/deathcurves_plot.r (hunk 1 above)
Comment (MikeLippincott) [reply figures/death_curves/scripts/deathcurves_plot.r:20]: removed

File: figures/death_curves/scripts/deathcurves_plot.r (hunk 2 above)
Comment (MikeLippincott) [reply figures/death_curves/scripts/deathcurves_plot.r:35]: removed this portion

**Pull request merged**